editor/project_manager.cpp

```diff
@@ -2015,6 +2015,10 @@ void ProjectManager::_confirm_update_settings() {
 }
 
 void ProjectManager::_open_selected_projects() {
+	// Show loading text to tell the user that the project manager is busy loading.
+	// This is especially important for the HTML5 project manager.
+	loading_label->set_modulate(Color(1, 1, 1));
```

Comment:
I don't see any code that would actually hide it again once it finished loading?

Reply:
The project manager quits as soon as the project is starting to be opened. Therefore, there should be no need to hide the text.